### DISCUSSION TITLE
Error updating whitespace and reformatting data to lowercase in Dbeaver.

### DISCUSSION THREAD

**Discussion by Maianh2510** [dbeaver/dbeaver#35334] [Q&A] [answered]
### Description

Hi everyone, I have a question: after I run the update command to remove spaces and convert the data to lowercase, but after I run the command, the result of the full name column failed. The names of all customers have been replaced with the first person's name. Where is the error in Dbeaver?
![5](https://github.com/user-attachments/assets/609d0b7c-d682-47e0-8f93-c20a1945566d)
![6](https://github.com/user-attachments/assets/01abeff8-c53d-496b-8bef-330fb02b6eda)

### DBeaver Version

Dbeaver 24.1.5

### Operating System

window 10

**Comment by TB-HunBoka** [comment, marked as answer]
This is not a Dbeaver issue but a querry issue. You told the query to replace each row with one string as the sub query will return a list of values and I guess the DB used the first value of the result set.
I am not a sqlite user, but I guess it should be this:
Set full_name = LOWER(LTRIM(full_name))
from ....

Or as in oracle you need to provide a joining condition between the value you would update and the value you would write in the field.

**Comment by Maianh2510** [comment]
@TB-HunBoka Thanks so much. I completed the mission thanks to you
